compiler/passes.nim

```diff
@@ -211,7 +211,7 @@ proc processModule*(graph: ModuleGraph; module: PSym, stream: PLLStream,
             if n.kind == nkEmpty: break
             sl.add n
           if sfReorder in module.flags:
-            sl = reorder sl
+            sl = reorder(sl, module.fileIdx.toFullPath)
```

Comment:
I think reorder should get 'module.fileIdx' instead and do the 'toFullPath' lazily.